homework_03_12_3/src/Animal.java

```diff
@@ -0,0 +1,31 @@
+
+public class Animal {
+    String habit;
+    boolean isFly;
+    boolean isMammals;
+
+    Animal(String habit,boolean isFly,boolean isMammals){
+        this.habit=habit;
+        this.isFly=isFly;
+        this.isMammals=isMammals;
+    }
+   String convert(boolean getConvert){
```

Comment:
getConvert is a bad example of variable name

Reply:
I don't know how to do it in main
I got mistakes